gnucash/gtkbuilder/window-autoclear.glade

```diff
@@ -68,8 +71,13 @@
               <object class="GtkLabel" id="label1">
                 <property name="visible">True</property>
                 <property name="can_focus">False</property>
-                <property name="label" translatable="yes">&lt;b&gt;Auto-Clear Information&lt;/b&gt;</property>
+                <property name="label" translatable="yes">&lt;b&gt;Auto-Clear Information&lt;/b&gt;
+Use this dialog if you want GnuCash to automatically find which transactions are cleared, given an ending balance. For example, said ending balance can be the current balance given by your online bank.
+
+&lt;b&gt;Note&lt;/b&gt;: The complexity of this algorithm is quite high (&lt;a href="https://en.wikipedia.org/wiki/Knapsack_problem#Computational_complexity"&gt;O(nW)&lt;/a&gt;). Use this only on a dozen of uncleared splits.</property>
                 <property name="use_markup">True</property>
+                <property name="wrap">True</property>
+                <property name="max_width_chars">80</property>
```

Comment:
Several translators had already translated the first line.
After a recent msgmerge I got for de.po:
```
[30855]
#: gnucash/gtkbuilder/window-autoclear.glade:74
msgid ""
"<b>Auto-Clear Information</b>\n"
"Use this dialog if you want GnuCash to automatically find which transactions "
"are cleared, given an ending balance. For example, said ending balance can "
"be the current balance given by your online bank."
msgstr ""

;
[31375]:
#~ msgid "<b>Auto-Clear Information</b>"
#~ msgstr "<b>Information zum automatischen Kontenabgleich</b>"
```
In that state the program would display english text only.
 
I wonder, why you keep the titlebar (line 9) empty? Shouldn't that be the first place for the title?
Alternatively you could split the text in multiple labels.

A few other notes from the translator view - although I know, it is not always easy in glade.

- Try to avoid markups.

- Ideally the lengh is one paragraph.
- more in https://wiki.gnucash.org/wiki/I18N and https://wiki.gnucash.org/wiki/GUI_Guidelines